server/models.py

```diff
@@ -0,0 +1,44 @@
+from app import db
```

Comment:
im pretty sure this would create a circular dependency which is a yucky  yucky error.

Reply:
Wanna just throw models in app.py?